website/src/pages/case-studies/workiva-batch-changes.tsx

```diff
@@ -0,0 +1,115 @@
+                        dozens of repositories, spend time improving their internal tool to help automate the process,
+                        or add the update to their backlog.
+                    </p>
+
+                    <InContentBlockquote
+                        quote="We fell into the habit of letting tech debt accumulate to the point where all of the sudden we’d have to bring everything to a screeching halt and do nothing for a month or sprint or even a quarter and clean up tech debt around a certain area. The easier the tooling, the faster we can release breaking changes."
+                        author={trentGrover}
+                    />
+
+                    <h2 className="pt-5 pb-1">
+                        Automating large-scale updates with Batch Changes
+                    </h2>
+
+                    <p>
+                        As an organization that values paying down tech debt, Workiva’s Client Platform team started using Sourcegraph Batch Changes to help them efficiently propagate updates to dependencies across all of their repositories without any ongoing maintenance. The team has already used Batch Changes to:
+                        <ul>
+                            <li>Propagate a new version of React to all frontend repos</li>
+                            <li>Update API versions of kubernetes resources</li>
+                            <li>Migrate to a new CDN while updating all code references</li>
+                            <li>Update UI component syntax as necessary to support a new version of the Dart language</li>
+                            <li>Run test batch changes to ensure that new chrome web browser releases wouldn't result in any software failures</li>
+                        </ul>
+                    </p>
+
+                    <InContentBlockquote
+                        quote="The fact that Batch Changes runs each step via Docker is super powerful. As long as there's a tool out there that does what you need, you can incorporate it into Batch Changes with minimal effort, and that's been awesome for us."
+                        author="Evan Weible, Staff Software Engineer, Workiva"
+                    />
+
+                    <h2 className="pt-5 pb-1">
+                        Workiva reduces the time it takes to make large-scale code changes by 80%
+                    </h2>
+
+                    <p>The Client Platform team has already used Batch Changes numerous times to propagate large-scale updates to the frameworks and shared libraries they maintain. In comparison to manually making these changes, Batch Changes reduces the time it takes to make large-scale code changes by 80%.</p>
+
+                    <InContentBlockquote
+                        quote="Updating all of our repositories with Batch Changes saves time, is less error-prone, and gives us confidence that everything is going to plan."
```

Comment:
@jeanduplessis - for some reason, the netlify preview shows the incorrect quote.